ddtrace/tracer/sqlcomment.go

```diff
@@ -83,6 +86,9 @@ func (c *SQLCommentCarrier) Inject(spanCtx ddtrace.SpanContext) error {
 			if v, ok := ctx.meta(ext.Version); ok {
 				version = v
 			}
+			if s := ctx.span; s != nil {
```

Comment:
Any reason to create `s` here? Perhaps just `if ctx.span != nil` ?

Reply:
Good catch. Updated ✅